core/src/main/java/org/apache/calcite/sql/SqlKind.java

```diff
@@ -429,6 +429,11 @@
    */
   POSIX_REGEX_CASE_INSENSITIVE,
 
+  /**
+   * The "REGEXP_SUBSTR" operator.
+   */
+  REGEXP_SUBSTR,
+
```

Comment:
We may not to add a `SqlKind` just for a function.

Reply:
+1.